src/components/Daos/DaosPage.tsx

```diff
@@ -98,7 +98,8 @@ class DaosPage extends React.Component<IProps, IState> {
     // Always show Genesis Alpha first
     let finalDAOList = allDAOs.filter((d: DAO) => d.staticState.name === "Genesis Alpha" && d.staticState.name.toLowerCase().includes(search));
 
-    if (process.env.NODE_ENV === "staging") {
+     // eslint-disable-next-line no-extra-boolean-cast
+    if (process.env.SHOW_ALL_DAOS === "true") {
```

Comment:
This should be removed.  It doesn't work and is done correctly in another PR

Reply:
it is doing what it suppose to do .

Reply:
but it will always return false.

Reply:
plus everything else I pointed out

Reply:
NO . it is return TRUE!
and it is working